src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(String color, double radius) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a variable for the `radius` and pass it to the constructor for better code readability.